acceptance/tests/sameness/sameness_test.go

```diff
@@ -127,19 +136,19 @@ func TestFailover_Connect(t *testing.T) {
 				+-------------------------------------------+
 			*/
 
-			members := map[string]*member{
-				keyPrimaryServer: {context: env.DefaultContext(t), hasServer: true},
-				keyPartition:     {context: env.Context(t, 1), hasServer: false},
-				keyPeer1:         {context: env.Context(t, 2), hasServer: true},
-				keyPeer2:         {context: env.Context(t, 3), hasServer: true},
+			testClusters := clusters{
+				keyCluster01a: {name: peerName1a, context: env.DefaultContext(t), hasServer: true, acceptors: []string{peerName2a, peerName3a}},
+				keyCluster01b: {name: peerName1b, context: env.Context(t, 1), partition: cluster01Partition, acceptors: []string{peerName2a, peerName3a}},
+				keyCluster02a: {name: peerName2a, context: env.Context(t, 2), hasServer: true, acceptors: []string{peerName3a}},
+				keyCluster03a: {name: peerName3a, context: env.Context(t, 3), hasServer: true},
 			}
 
 			// Setup Namespaces.
-			for _, v := range members {
+			for _, v := range testClusters {
 				createNamespaces(t, cfg, v.context)
 			}
 
-			// Create the Default Cluster.
+			// Create the cluster-01-a.
```

Comment:
This is a great idea, and will require some thought. To keep in scope, I discussed with Curt to table this for now. I created a JIRA ticket to track as this might help other mult-cluster tests as well https://hashicorp.atlassian.net/browse/NET-5224